.github/workflows/examples-nrfconnect.yaml

```diff
@@ -45,12 +45,14 @@ jobs:
             #   with:
             #       languages: "cpp, python"
             #       queries: +security-and-quality
-            - name: Build example nRF Connect SDK Lock App
-              run: scripts/examples/nrfconnect_example.sh lock-app
-            - name: Build example nRF Connect SDK Lighting App
-              run: scripts/examples/nrfconnect_example.sh lighting-app
-            - name: Build example nRF Connect SDK Shell
-              run: scripts/examples/nrfconnect_example.sh shell
+            - name: Build example nRF Connect SDK Lock App on nRF52840 DK
+              run: scripts/examples/nrfconnect_example.sh lock-app nrf52840dk_nrf52840
+            - name: Build example nRF Connect SDK Lighting App on nRF52840 DK
+              run: scripts/examples/nrfconnect_example.sh lighting-app nrf52840dk_nrf52840 -DOVERLAY_CONFIG=rpc.overlay
```

Comment:
I would say, we should build the Lighting app without overlay as well as it's the default configuration.

Reply:
Ok, I misunderstood that we want to run builds with overlay by default, fixed.